resources/lang/pt-br/users.php

```diff
@@ -144,36 +145,36 @@
             'country_age' => ':age anos de :country.',
         ],
         'page' => [
-            'description' => '<strong>eu!</strong> é uma área pessoal customizável na sua página de perfil.',
-            'edit_big' => 'Edite-me!',
+            'description' => '<strong>eu!</strong> é uma área pessoal personalizável na sua página de perfil.',
+            'edit_big' => 'Editar eu!',
             'placeholder' => 'Digite o conteúdo da página aqui',
             'restriction_info' => "Você precisa ser um <a href='".osu_url('support-the-game')."' target='_blank'>osu!supporter</a> para desbloquear essa função.",
         ],
         'plays_with' => [
             '_' => 'Joga com',
             'keyboard' => 'Teclado',
             'mouse' => 'Mouse',
-            'tablet' => 'Mesa digitalizadora',
-            'touch' => 'Touch Screen',
+            'tablet' => 'Tablet',
+            'touch' => 'Tela sensível ao toque',
```

Comment:
if you want to keep it translated to portuguese you should also translate "tablet" as well. "Mesa digitalizadora" is the correct name for "tablet".